shared/login/join-or-login/index.tsx

```diff
@@ -15,47 +15,49 @@ type Props = Kb.PropsWithTimer<{
 const Intro = (props: Props) => {
   Kb.useInterval(props.checkIsOnline, 2000)
   return (
-    <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} alignItems="center">
-      <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.header}>
-        <InfoIcon />
-      </Kb.Box2>
-
-      {!!props.bannerMessage && (
-        <Kb.Box2 direction="vertical" fullWidth={true} style={styles.banner}>
-          <Kb.Text center={true} type="BodySmallSemibold" style={styles.bannerMessage}>
-            {props.bannerMessage}
-          </Kb.Text>
+    <SignupScreen noBackground={true} title="Log in foo">
```

Comment:
????

Reply:
why does this have 'log in foo' in the title?

Reply:
I think you're commenting on something outdated.